src/inference/src/dev/core_impl.cpp

```diff
@@ -743,13 +743,13 @@ ov::Plugin ov::CoreImpl::get_plugin(const std::string& pluginName) const {
                 // the same extension can be registered multiple times - ignore it!
             }
         } else {
-            try_to_register_plugin_extensions(desc.libraryLocation);
+            try_to_register_plugin_extensions(desc.libraryLocation.native());
         }
 
         return plugins.emplace(deviceName, plugin).first->second;
     } catch (const ov::Exception& ex) {
         OPENVINO_THROW("Failed to create plugin ",
-                       ov::util::from_file_path(desc.libraryLocation),
+                       desc.libraryLocation.string(),
```

Comment:
```suggestion
                       desc.libraryLocation,
```
The operator<< should call string conversion.

Reply:
Done